.ci/main-trigger/Jenkinsfile

```diff
@@ -0,0 +1,50 @@
+// https://github.com/camunda/jenkins-global-shared-library
+// https://github.com/camunda/cambpm-jenkins-shared-library
+@Library(['camunda-ci', 'cambpm-jenkins-shared-library@3021']) _
+
+pipeline {
+  agent {
+    node {
+      label 'jenkins-job-runner'
+    }
+  }
+  environment {
+    LOGGER_LOG_LEVEL = 'DEBUG'
+  }
+  options {
+    buildDiscarder(logRotator(numToKeepStr: '5'))
+    disableConcurrentBuilds()
+  }
+  triggers {
+    pollSCM('H H(16-17) * * *') // CE main daily trigger
+  }
+  stages {
+    stage('Trigger Main EE') {
+      when {
+        branch cambpmDefaultBranch();
+      }
+      steps {
+        script {
+          // JOB_NAME, e.g.: '7.15/cambpm-ce/cambpm-main/7.15'
+          // keep leading slash for the absolute project path
+          platformVersionDir = "/" + env.JOB_NAME.split('/')[0]
+
+          // build quietPeriod: 10, wait: false, job: platformVersionDir + "/cambpm-ce/cambpm-main/"+ cambpmDefaultBranch()
```

Comment:
Do we need this?
```suggestion
```

Reply:
I'll remove it, it was a hint if we don't use the custom call.